src/minisweagent/environments/singularity.py

```diff
@@ -25,6 +25,8 @@ class SingularityEnvironmentConfig:
     """Path to the singularity executable."""
     sandbox_build_retries: int = 3
     """Number of retries for building the sandbox if an error occurs."""
+    writeable_tmp: bool = False
```

Comment:
Why is this needed? Instead of writing the eval script to a file shared with the host, can we just run it via `.execute`?